spec/features/player_name_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'form submit' do
```

Comment:
Nice and simple first test to check your names are outputting on the page for the user.